__tests__/account.test.ts

```diff
@@ -421,6 +421,10 @@ describe('deploy and test Wallet', () => {
     const signature = await account.signMessage(typedDataExample);
     const verifMessageResponse: boolean = await account.verifyMessage(typedDataExample, signature);
     expect(verifMessageResponse).toBe(true);
+    const messageVerificationFunctionName = account.signatureVerifFunctionName;
+    expect(
+      ['isValidSignature', 'is_valid_signature'].includes(messageVerificationFunctionName!)
+    ).toBe(true);
```

Comment:
Could also use the `toContain()` matcher: `expect([...]).toContain(functionName)`.

Reply:
Do you want it to be modified?